src/api/api.ts

```diff
@@ -62,3 +62,41 @@ export const hentTilretteleggingsbehov = async (): Promise<Respons> => {
         };
     }
 };
+
+type SuksessOppfolgingsstatus = {
+    status: Status.Suksess;
+    oppfolgingsstatus: Oppfølgingsstatus;
+};
+
+export type ResponsOppfolgingsstatus =
+    | IkkeLastet
+    | LasterInn
+    | SuksessOppfolgingsstatus
+    | Feil
+    | UkjentFeil;
+
+export const hentOppfolgingsstatus = async (): Promise<ResponsOppfolgingsstatus> => {
+    try {
+        const respons = await fetch('/person/behov-for-tilrettelegging/oppfolgingsstatus', {
+            method: 'GET',
+            credentials: 'include',
+        });
+
+        if (respons.ok) {
+            const oppfolgingsstatus: Oppfølgingsstatus = await respons.json();
+
+            return {
+                status: Status.Suksess,
```

Comment:
Vi kan evt droppe json bo i dette grensesnittet, og bruke 404 på oppfølgingsstatus som mangler. Blir enklere og penere kode av det.

Reply:
"Droppe json bo"? Mener du at 200 = er under oppfølging og 404 = er ikke under oppfølging? Høres bra ut, blir jo litt enklere kode når vi slipper å typesette oppfølging og dens status

Reply:
Responsen fra apiet er jo json med en verdi true/false, så kan vel ikke bruke 404 eller 200 for å sjekke dette...